rules/manager.go

```diff
@@ -76,18 +76,22 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 				Help:       "The duration for a rule to execute.",
 				Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
 			}),
-		evalFailures: prometheus.NewCounter(
+		evalFailures: prometheus.NewCounterVec(
```

Comment:
It makes sense to move them below in the file.

Reply:
What do you think if I update all `CounterVec` and `GaugeVec` to `Counter` and `Gauge`? Since in one group metrics, their `rule_group` label is the same